### ISSUE TITLE
WG bugg, no raid groups

### ISSUE THREAD

**Issue by Ukhozii** [WoWManiaUK/Redemption#1617]
Hello, basicly there were no raid groups. Only a few on Horde's side counted as in WG the rest didn't.
Those "in" WG were not in the same raid, they could create their own party (however, those in the party didn't get entered to WG, simply easier to talk to eachother)
The ones "in" WG got around 20 tenancy each as it counted as if they were there on their own. (around 4 of us were "in" WG, the rest of us helped as much as we could) The rest seemed normal on their side except the alone part.
The rest of us that weren't counted as "in" WG didn't get tenancy, no rank and it was as if we weren't in WG and as it normaly is outside of WG (except that we did get PVP tagged) and no emblems at the end, Also no message that told us that WG was finnished, I could see the counter telling us how much time was left.

Ok, so the second time (2:nd WG) we got a small raid group that some of us got entered to. I do not know how many alliance there were (saw like 3) but we got 6 tenancy. The Alliance players could simply have been somewhere else. 

Both times the alliance had WG, we won the last one. So we will see who ends up with the bugg, if the problem still remains that is

Problem remains, the 3:rd time I joined some got entered as they should, and a lot of us didn't. They could only help but get no rank and no emblems. We tried entering them into our raid but the problem remained even though they were in the same raid group as us. They couldn't get any ranks and no buffs nor did they get any emblems at the end

Also, flying in to WG instead or the teleport seems to solve the problem for some
squalllh1989 has more info. Also Ally seems to have problems, I have no more info about the Alliance side of this

Regards Ukhozi

**Comment by fvicaria** [comment]
You cannot join battle after it has started.

**Comment by Ukhozii** [comment]
it's WG, Winter grasp, everyone vs everyone and you can always join that one even if it's not 30 min left but 30 sec

**Comment by squalllh1989** [comment]
acually, you could join the battle after it started.
source: http://wowwiki.wikia.com/wiki/Wintergrasp

Wrath-Logo-Small Patch 3.2.0 (04-Aug-2009):
To provide players with a more transparent notification of when Wintergrasp battles occur, as well as better control zone population and stability, several changes have been made:
- Players now have the option to queue for Wintergrasp from a Wintergrasp Battlemaster in any capital city or by simply entering the Wintergrasp zone.
**- Queuing will begin 15 minutes before each battle. If chosen, you will automatically be teleported to the zone. Any players in the zone who have not been chosen from the queue will be teleported out when the battle begins.**
**- The queue system remains active for the entire battle. As soon as a player leaves, a new one will be chosen from the queue.**
- Trying to enter Wintergrasp during an active battle for which you have not been chosen will teleport you out. Please note that, as you are now able to fly over Wintergrasp, you will only be teleported out if you try to land and join the battle.
- Level 80 players get higher priority in the queue than lower level players. In addition, a random selection of queued players will be taken from both the Battlemasters and the zone itself.
- Level 75 is required to queue
**- The queue will accept up to 120 players from each faction, resulting in a maximum battle of 240 players at a time.**

Based on the 3 line i marked bold, as our wintergasp haven't ever reach the limit of 120 player per faction, as soon as you get in the queue at the capital city / dalaran or by fly in, got asked and accept to queue, you will get chosen to join the battle right away, while decline the queue while inside wintergasp when the battle begin teleport you out of it.

**Comment by squalllh1989** [comment]
He is what i investigated about the problem that Ukhozii reported:
- I joined the WG battle start at 14:00 ST on horde side, by talk to Magister Surdiel <Wintergrasp Battle-Mage> (http://www.wow-mania.com/armory/?npc=32170) in dalaran, accept the queue and accept the call to join the battle, i got ported in the battle and joined the raid normally (perfect).
- After that, i decided to leave the battle throught the portal inside the vault to violet's hold, i tried to get in by the Defender's portal in front of the WG fortress door, i got about a few loading screen apear (weird, normally, i will just get port in the vault) and then i got port out of WG, just like i decline the WG queue or afk inside WG, but i'm still in the WG raid.
- I leave the WG raid and try to fly down to WG again, but there is no pop up ask me to join the battle of WG (normally, and before, there is), i also didn't get kicked out of WG like normally it will be.
- I heartstone back to dalaran, and this time try to teleport to WG throught the WG portal near Magister Surdiel. I got ported to the fort (horde is controlling wg, so we are on defence side) like normal, but there isn't a pop up ask me to join WG battle as i enter when the battle is going on, and it didn't kick me out also.

What it should work:
- When you enter WG by fly there, use the portal to teleport there, or you are already inside WG when the battle start or are going on, there should alway a pop up ask you to join the battle. If you accept, you join the WG raid, and fight, if you decline, you got port out of it, to the zone decided.
- When you talk to the NPC to queue for WG battle 15 min before the battle start, you are asked to join the WG queue, whenn you accpep, you join the queue, when the battle start, you are asked to join the battle, if you accept you will get ported to the default zone of your faction (depend on you are on attack or defence side).
- If you are on defence side, the defender's portall should allow you to port inside the vault anytime you want.
- If you leave WG by run out of it, use the portal to get out, mage teleport, heartstone, etc, you will leave the raid, your rank got reset, and you get a message "you have exited the battle for WG fortress".
All of these was working perfectly before, so this must be newly bugged.
